### PR TITLE
Trainee mainpage

### PR DESCRIPTION
**Pull request by codegracie**
Add User Homepage, profile, course page and navigation

### CODE REVIEW

**Comment by kian1213** [comment]
ok

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,29 +1,55 @@
...
+              </a>
+              <ul class="dropdown-menu">
+                <li><%= link_to t(".tablink_profile"), current_user %></li>
+                <li><%= link_to t(".tablink_setting"), "#" %></li>
+                <li class="divider"></li>
+                <li><%= link_to t(".tablink_logout"), destroy_user_session_path, method: "delete" %></li>
+              </ul>
+            </li>
+          </nav>
+          <% else %>
```
Comment (MuhammadTamzid) [inline app/views/layouts/_header.html.erb]: indent

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -28,3 +28,4 @@
 
 
 
+
```
Comment (dieunb) [inline app/assets/stylesheets/application.scss]: Checkout this unchange file

File: app/assets/stylesheets/layouts/_header.scss [hunk 3]
```diff
@@ -1,2 +1 @@
-
```
Comment (dieunb) [inline app/assets/stylesheets/layouts/_header.scss]: Checkout this unchange file

File: app/views/shared/_user_activities.html.erb [hunk 4]
```diff
@@ -1 +1,3 @@
 <h1><%= t ".title" %></h1>
+
```
Comment (dieunb) [inline app/views/shared/_user_activities.html.erb]: Checkout this unchange file

File: app/views/shared/_user_profile.html.erb [hunk 5]
```diff
@@ -1,11 +1,24 @@
...
-  <div class="row">
-    <h3><%= @user.full_name %></h3>
-  </div>
-  <div class="row">
-    <%= @user.user_type %>
-  </div>
-</div>
\ No newline at end of file
+</div>
+
```
Comment (dieunb) [inline app/views/shared/_user_profile.html.erb]: remove any white line

File: app/assets/stylesheets/layouts/_header.scss (hunk 3 above)
Comment (codegracie) [inline app/assets/stylesheets/layouts/_header.scss]: please guide. i have already modified this file. please check again. thnks

File: app/assets/stylesheets/layouts/_footer.scss [hunk 6]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (dieunb) [inline app/assets/stylesheets/layouts/_footer.scss]: Checkout this unchange file

**Pull request merged**